File: c++/q_2.cpp
#include <iostream>
#include <string.h>
using namespace std;

std::string hmilu(int petals){
    int new_petals = petals %6;
    switch (new_petals)
    {
    case 0 /* constant-expression */:
        /* code */
        return "Not at all";
    case 1:
        return "I love you";
    case 2:
        return "A little";
    case 3:
        return "A lot";
    case 4: 
        return "Passionately";
    case 5:
        return "Madly";

    default:
        break;
    }
}

int main(){
    int petals;
    std::cout<<"Please enter a value to get know how much you love him/her: ";
    std::cin>>petals;
    string HMILU = hmilu(petals);
    std::cout<<"The result is: "<< HMILU;

}

File: python/q_2.py
def hmilu(petals):
    new_petals = petals %6
    if new_petals == 0:
        return "Not at all"
    elif new_petals == 1:
        return "I love you"
    elif new_petals == 2:
        return "A little"
    elif new_petals == 3:
        return "A lot"
    elif new_petals == 4:
        return "Passionately"
    else:
        return "Madly"


if __name__ == "__main__":
    petals = int(input("Enter some value to get know how much you love him/her: "))
    result = hmilu(petals)
    print(result)